src/modules/filelist/virtualized/cells/SharingShortcutBadge.jsx

```
@@ -2,22 +2,26 @@ import PropTypes from 'prop-types'
 
 const SharingShortcutBadge = ({ file }) => {
   const { t } = useI18n()
 
   if (isSharingShortcutNew(file)) {
     return (
-      <Circle size="xsmall" backgroundColor="var(--errorColor)">
+      <Avatar
+        color="var(--errorColor)"
+        textColor="var(--white)"
+        size="xs"
+      >
```

ESLint

[error] Replace `⏎········color="var(--errorColor)"⏎········textColor="var(--white)"⏎········size="xs"⏎······` with `·color="var(--errorColor)"·textColor="var(--white)"·size="xs"`

(prettier/prettier)